tests/common/CacheWithErrors.h

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Can you merge it into BaseCacheWrapper.h (or how it should be actually called, KeyValueCacheTestable).